kubernetes/kubernetes.checkout

```diff
@@ -2,4 +2,5 @@ staging/src/k8s.io/api/core/v1/generated.proto
 staging/src/k8s.io/apimachinery/pkg/api/resource/generated.proto
 staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto
 staging/src/k8s.io/apimachinery/pkg/runtime/generated.proto
+staging/src/k8s.io/apimachinery/pkg/runtime/schema/generated.proto
```

Comment:
I probably messed up when I edited this file last, because this should have been there despite the actual source not being used at runtime (it is required at generation time)